cli/lightning-cli.c

```diff
@@ -145,6 +145,7 @@ static char *opt_set_ordered(enum input *input)
 static bool is_literal(const char *arg)
 {
 	size_t arglen = strlen(arg);
+	if(!arglen) return false;
```

Comment:
Trivial style things:
1. space after if.
2. it's genuinely a number, so I prefer (arglen == 0).
3. I dislike single-line ifs, especially with control flow.

Thanks!